FlightReservation/src/helpers/firebaseSignUp.js

```diff
@@ -12,6 +12,7 @@ export const signInWithNameEmailAndPassword = (name, email, password) => {
         user
           .updateProfile({displayName: name})
           .then(() => resolve('User created & signed in'));
+        console.log('Signed Up!');
```

Comment:
Remove your console.log () when uploading code to your remote repository.

Reply:
Deleted